### ISSUE TITLE
Logos keep reverting to use my public IP

### ISSUE THREAD

**Issue by my-ugly-code** [xteve-project/xTeVe#345]
**Describe the bug**
Channel logos in Mapping section are using my public IP address. I have backups that contain xepg.json with correct urls, but restoring those backups doesn't seem to change the urls back to the working (local server IP) url. 

**Server (please complete the following information):**
- Windows 10
- 64 bit
- xTeVe Version 2.2.0 Build: 0200, DB v2.1.0

**To Reproduce**
1. Go to Mapping tab
2. Click on a channel
3. Edit logo url by uploading a logo
4. Click Done to close MAPPING popup
5. Go to Settings tab
6. Click Restore and select any backup that uses different values for logo url set in step 3.
7. Allow restore to complete, Save
8. Go back to mappings tab and see if channel logo url changed

**Expected behavior**
I expected the restore to pull in the backups xepg.xml and use the tvg-logo url from the backup selected. It seems to not change at all. I expected that manually set channel logos would stay and not be overwritten from my source.xml. Also, this (the overwrite) happens to all channel logos that I manually set, not just a subset.

**Desktop (please complete the following information):**
 - Same as server

**Additional context**
I also tried stopping the xTeVe service, then manually overwriting my (broken) C:\Users\<me>\.xteve\xepg.xml with the (good) backup file contents. When I start xTeVe back up, the C:\Users\<me>\.xteve\xepg.xml reverts back to version using my external IP. My source epg.xml contains a few <channel><icon-src> values, but not all. Did I break something? I'm hoping to avoid manually setting all 400 channel logos again, as they're set correctly in several backup files. Standing by to provide any further details.

**Comment by MCTyler** [comment]
Sounds like you enabled "Caching of Images"
![image](https://user-images.githubusercontent.com/6390898/200799114-9290fe1e-ba12-4dfa-8f0c-719ed8d51e2f.png)
![image](https://user-images.githubusercontent.com/6390898/200799253-fe6723b1-8fab-4827-ba40-80a353fa1c14.png)

**Comment by my-ugly-code** [comment]
Thanks so much for your reply! I was able to get my images back. I think the root cause of the issue I reported can just be me :) and not knowing how to edit a file within a .zip folder. I noticed that while saving an ip find/replace'd xepg.xml from within a .zip, Sublime was always warning me it was a 'protected file', but appeared to let me "save" it anyway without errors...appearances can be deceiving. This morning, with xTeVe service stopped, I did the same find/replace, "save", got the 'protected file' warning, "saved" anyway, then I closed and reopened the file...sure enough, the changes I made were not being saved. To my own detriment, I was never confirming the save actually happened before, I'd just restart xTeVe, see logos missing, _then_ open the xepg file and see what I thought were overwrites.

The public ip's could have been introduced during 'Caching of Images', which I experimented with in the past, but could have been something completely unrelated to xTeVe...that part remains a bit of a mystery. Regardless, super happy I don't have to set all the images manually again, and if faced with the similar self-induced issues in the future, I'll know how to fix them now. I'll mark this as **closed - user error** :) . Sorry for taking your time, but thank you again!
